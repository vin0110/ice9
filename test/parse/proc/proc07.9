proc a(b : int, c: bool)
  g();
end
