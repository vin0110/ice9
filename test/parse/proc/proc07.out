PROC: a
(return)
  nil
(body)
  CALL:
    ID:'g'
  (arglist)
    nil
